### PR TITLE
Implement IApplicationCreator & IApplicationAccessor & IAppletAccessor

### PR DESCRIPTION
**Pull request by greggameplayer**

### CODE REVIEW

**Comment by greggameplayer** [comment]
I think it's ready to merge

**Comment by gdkchan** [comment]
A bunch of functions are just returning an `IApplicationAccessor`, this looks wrong, afterall why would they have a bunch of functions all doing the same thing?

**Comment by greggameplayer** [comment]
see here : http://switchbrew.org/index.php?title=AM_services#IApplicationCreator

File: Ryujinx.Core/OsHle/Services/Am/IApplicationCreator.cs [hunk 1]
```diff
@@ -13,8 +13,39 @@ public IApplicationCreator()
         {
             m_Commands = new Dictionary<int, ServiceProcessRequest>()
             {
-                //...
+                { 0,   CreateApplication                      },
+                { 1,   PopLaunchRequestedApplication          },
+                { 10,  CreateSystemApplication                },
+                { 100, PopFloatingApplicationForDevelopment   }
```
Comment (gdkchan) [inline Ryujinx.Core/OsHle/Services/Am/IApplicationCreator.cs]: Please align the `}` properly as theres a bunch of unneeded spacing.

**Comment by greggameplayer** [comment]
and it say that here too : https://reswitched.github.io/SwIPC/ifaces.html#nn::am::service::IAllSystemAppletProxiesService

File: Ryujinx.Core/OsHle/Services/Am/IAppletAccessor.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        public IAppletAccessor()
+        {
+            m_Commands = new Dictionary<int, ServiceProcessRequest>()
+            {
+                {  0, GetAppletStateChangedEvent },
+                {  1, IsCompleted                },
+                { 10, Start                      },
+                { 20, RequestExit                },
+                { 25, Terminate                  },
+                { 30, GetResult                  },
```
Comment (gdkchan) [inline Ryujinx.Core/OsHle/Services/Am/IAppletAccessor.cs]: Comma is not necessary on the end of this last entry.

**Comment by greggameplayer** [comment]
@gdkchan It's good like this ?

**Comment by greggameplayer** [comment]
and also what method can i insert in the function InitializeGamePlayRecording in order to takes a TransferMemory Handler ?

**Comment by gdkchan** [comment]
You just still add a single space before the `}`. Just see how it looks like on the other services.

**Comment by greggameplayer** [comment]
@gdkchan I think it's good now

**Comment by Cyuubi** [comment]
looks good to me 👍

**Comment by greggameplayer** [comment]
I close that PR i will reopen a new one for that

**Pull request closed without merging**